ts/webui/src/components/overview/table/SuccessTable.tsx

```diff
@@ -60,32 +90,51 @@ class SuccessTable extends React.Component<SuccessTableProps, SuccessTableState>
             }
         });
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        const newItems = this.copyAndSort(source, currColumn.fieldName!, currColumn.isSortedDescending);
+        const newItems = copyAndSort(source, currColumn.fieldName!, currColumn.isSortedDescending);
         this.setState({
             columns: newColumns,
-            source: newItems
+            source: newItems,
+            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+            sortInfo: { field: currColumn.fieldName!, isDescend: currColumn.isSortedDescending }
         });
     };
 
-    private copyAndSort<T>(items: T[], columnKey: string, isSortedDescending?: boolean): T[] {
-        const key = columnKey as keyof T;
-        return items.slice(0).sort((a: T, b: T) => ((isSortedDescending ? a[key] < b[key] : a[key] > b[key]) ? 1 : -1));
-    }
-
-    tooltipStr = (
+    private tooltipStr = (
         <React.Fragment>
             The experiment is running, please wait for the final metric patiently. You could also find status of trial
             job with <span>{DETAILTABS}</span> button.
         </React.Fragment>
     );
 
-    columns = [
+    private columns = [
+        {
+            key: '_expand',
+            name: '',
+            onRender: (item: any): any => (
+                <Icon
+                    aria-hidden={true}
+                    iconName='ChevronRight'
+                    styles={{
+                        root: {
+                            transition: 'all 0.2s',
+                            transform: `rotate(${this.state.expandRowIdList.has(item.id) ? 90 : 0}deg)`
+                        }
+                    }}
+                    className='cursor'
+                    onClick={this.expandTrialId.bind(this, Event, item.id)}
+                />
+            ),
+            fieldName: 'expand',
+            isResizable: false,
+            minWidth: 20,
```

Comment:
Is it possible to use the classname to control the column min-width and max-width?

Reply:
by design, minWidth is required filed on IColumn interface

![image](https://user-images.githubusercontent.com/61399850/99924538-bb326e00-2d75-11eb-8b53-f4f1f4d57a1d.png)